odml/property.py

```diff
@@ -247,6 +247,9 @@ def _validate_values(self, values):
             try:
                 dtypes.get(v, self.dtype)
             except Exception:
+                raise ValueError("odml.Property.values: passed value \'%s\' are not of "
```

Comment:
As stated in the issue, the `Format` part of the message should be only displayed in case of `date`, `time` and `datetime`. For the other data types the format message is not really useful.